precompile/fee_config_manager.sol

```diff
@@ -0,0 +1,44 @@
+// (c) 2022-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+interface FeeConfigManagerInterface {
```

Comment:
nit: is there some abstraction in solidity that allows us to inherit the `Admin` interface? like `is AllowList`?

Reply:
yes we use it in. https://github.com/ava-labs/subnet-evm/blob/fee-config-precompile/contract-examples/contracts/IFeeManager.sol#L5

I think in these files we wanted to explicitly show which functions are available for precompiled contracts.